src/main/steps/eligibility/lived-uk/content.test.ts

```diff
@@ -1,68 +1,85 @@
+import { YesNoNotsure } from '../../../app/case/definition';
+import { FormContent, FormFields, FormOptions } from '../../../app/form/Form';
 import { isFieldFilledIn } from '../../../app/form/validation';
 import { CommonContent, generatePageContent } from '../../common/common.content';
 
 import { generateContent } from './content';
+
 jest.mock('../../../app/form/validation');
 
-/* eslint-disable @typescript-eslint/ban-types */
-describe('lived-uk content', () => {
-  const commonContent = { language: 'en' } as CommonContent;
+const enContent = {
+  section: 'Eligibility to apply to adopt',
+  label:
+    'Have you and any other applicant if relevant, lived in the UK, Channel Islands or Isle of Man for at least 12 months (habitually resident)?',
+  livedUKNo: 'You cannot apply to adopt a child unless you have a permanent home here.',
+  moreInfo: 'More about adoption',
+  errors: {
+    livedUKEligible: {
+      required: 'Please answer the question',
+    },
+  },
+};
+
+const cyContent = {
+  section: 'Eligibility to apply to adopt (in Welsh)',
+  label:
+    'Have you and any other applicant if relevant, lived in the UK, Channel Islands or Isle of Man for at least 12 months (habitually resident)? (in welsh)',
+  livedUKNo: 'You cannot apply to adopt a child unless you have a permanent home here. (in welsh)',
+  moreInfo: 'More about adoption (in welsh)',
+  errors: {
+    livedUKEligible: {
+      required: 'Please answer the question (in welsh)',
+    },
+  },
+};
+
+/* eslint-disable @typescript-eslint/ban-types, @typescript-eslint/no-explicit-any */
+describe('eligibility > lived-uk > content', () => {
+  const commonContent = generatePageContent({
+    language: 'en',
+    userCase: {},
+  }) as CommonContent;
+
+  let generatedContent;
+  beforeEach(() => {
+    generatedContent = generateContent(commonContent);
+  });
+
   test('should return correct english content', () => {
-    const generatedContent = generateContent(commonContent);
-    expect(generatedContent.label).toEqual(
-      'Have you and any other applicant if relevant, lived in the UK, Channel Islands or Isle of Man for at least 12 months?'
-    );
-    expect(generatedContent.section).toEqual("Check you're eligible to adopt");
-    expect(generatedContent.livedUKNo).toEqual(
-      'You cannot apply to adopt a child unless you have a permanent home here.'
-    );
-    expect(generatedContent.one).toEqual('Yes');
-    expect(generatedContent.two).toEqual('No');
-    expect(generatedContent.hint).toEqual('You can answer yes if your permanent home is here.');
+    expect(generatedContent.section).toEqual(enContent.section);
+    expect(generatedContent.label).toEqual(enContent.label);
+    expect(generatedContent.livedUKNo).toEqual(enContent.livedUKNo);
+    expect(generatedContent.moreInfo).toEqual(enContent.moreInfo);
+    expect(generatedContent.errors).toEqual(enContent.errors);
```

Comment:
whilst you are here, can you simplify this using https://github.com/hmcts/adoption-web/blob/05ddc4ab44fa2e4bee1632469dc6df594ee7ca22/src/main/steps/sibling/placement-order-check-your-answers/content.test.ts#L51